src/lib/components/tablev2/SingleSelectableContent.tsx

```diff
@@ -56,6 +57,7 @@ export function SingleSelectableContent({
   selectedId,
   onRowSelected,
   customItemKey,
+  hasScrollbar: tableHasScrollbar,
```

Comment:
Maybe we also need the props for MultiSelectableTable as well for a sake of consistency